content/en/docs/setup/independent/ha-topology.md

```diff
@@ -0,0 +1,68 @@
+---
+reviewers:
+- sig-cluster-lifecycle
+title: Options for Highly Available Topology
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+
+This page explains the two options for configuring the topology of your highly available (HA) Kubernetes clusters.
+
+You can set up an HA cluster:
+
+- With stacked control plane nodes, where etcd nodes are colocated with control plane nodes
+- With external etcd nodes, where etcd runs on separate nodes from the control plane
+
+You should carefully consider the advantages and disadvantages of each topology before setting up an HA cluster.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Stacked etcd topology
```

Comment:
reproposing comment after reshuffle
I think we should clearly state that in this document we are considering stacked etcd cluster automatically created by kubeadm init and join.

Otherwise also a systemd managed etcd/an etcd cluster created with kubeadm phases installed on top of the control-plane machines qualifies as stacked etcd, but most of the following consideration about component coupling probably does not apply

@timothysc opinions?

Reply:
> considering stacked etcd cluster automatically created by kubeadm init and join.
added a note about that in the new `ha-topology` page.